CODEOWNERS

```diff
@@ -0,0 +1,71 @@
+# Scheduler
+master/internal/scheduler/    @aaron276h @rb-determined-ai
+
+# Provisioner
+master/internal/provisioner/  @shiyuann
+
+# Model registry
+# @eecsliu @katport
```

Comment:
@eecsliu @katport is there a main set of files for this? I found this stuff distributed across many parts of the code